contracts/dao/voting/vesting-lp-vault/README.md

```diff
@@ -0,0 +1,3 @@
+### Neutron Vesting LP Voting Vault
+
+This contract is not really a voting vault. It's rather an interface to get voting power from a Vesting LP contract. It's not possible to Bond or Unbond funds to this vault cause these ExecuteMsg handlers are introduced just to make the contract comply with the voting vault interface.
```

Comment:
What's the "Vauting vault" then huh?

Reply:
I believe the logic behind this statement is that there is no bonding for this contract, and a vault is something that actually keeps possessions

Reply:
I have copy pasted this documentation from lockdrop vault. And, to be fair, this documentation makes sense for me, since vaults usually allow you to deposit funds in order to receive voting power, but this vaults only calculate your voting power according to some external sources.

Reply:
According to our (recently updated hehe) doc, that's a virtual voting vault: https://docs.neutron.org/neutron/dao/overview#voting-vaults
However, we'll fix it in separate PR